app/routes/announcements/components/AnnouncementsCreate.tsx

```diff
@@ -48,136 +86,97 @@ let AnnouncementsCreate = ({
       {actionGrant.includes('create') && (
         <Flex column>
           <h1>Ny kunngjøring</h1>
-          <Form className={styles.form}>
-            <Field
-              className={styles.msgField}
-              name="message"
-              component={TextArea.Field}
-              placeholder="Skriv din melding her ..."
-              label="Kunngjøring:"
-            />
-            <span className={styles.formHeaders}>Mottakere:</span>
-            <Flex className={styles.rowRec}>
-              <Field
-                className={styles.recField}
-                name="users"
-                placeholder="Brukere"
-                filter={['users.user']}
-                isMulti
-                component={SelectInput.AutocompleteField}
-              />
-              <Field
-                name="groups"
-                placeholder="Grupper"
-                filter={['users.abakusgroup']}
-                isMulti
-                component={SelectInput.AutocompleteField}
-              />
-            </Flex>
-            <Flex className={styles.rowRec}>
-              <Field
-                className={styles.recField}
-                name="events"
-                placeholder="Arrangementer"
-                filter={['events.event']}
-                isMulti
-                component={SelectInput.AutocompleteField}
-              />
-              <Field
-                name="meetings"
-                placeholder="Møter"
-                filter={['meetings.meeting']}
-                isMulti
-                component={SelectInput.AutocompleteField}
-              />
-            </Flex>
-            <span className={styles.formHeaders}>Send på vegne av:</span>
-            <Field
-              name="fromGroup"
-              placeholder="Fra gruppe"
-              filter={['users.abakusgroup']}
-              component={SelectInput.AutocompleteField}
-            />
-
-            <Flex wrap>
-              <Button
-                onClick={handleSubmit((values) => onSubmit(values, false))}
-                disabled={disabledButton}
-              >
-                Opprett
-              </Button>
-              <Button
-                onClick={handleSubmit((values) => onSubmit(values, true))}
-                disabled={disabledButton}
-              >
-                Opprett og send
-              </Button>
+          <LegoFinalForm
+            validate={validate}
+            onSubmit={onSubmit}
+            initialValues={initialValues}
+            validateOnSubmitOnly
```

Comment:
Why do we use `validateOnSubmitOnly`? I like live feedback

Reply:
There's a weird bug when using `form.reset()` where it will validate all fields on submit, and since the user is not redirected here after submitting there'll instead be a bunch of validation warnings. Getting warnings after a successful request is bad feedback, so I reckoned it was better to only validate when actually submitting.

Reply:
Also,

> I like live feedback

Yes sure, me too, but the current live validation is a bit "aggressive" imo. Simply clicking on a field shouldn't make a huge red box appear. As long as it is clear that the field is required (e.g. with a red *), I see nothing wrong with only validating on submit. I'd say that's a pretty common approach